packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -594,6 +594,83 @@ describe('I18n', () => {
     });
   });
 
+  describe('#formatCurrencyExplicit()', () => {
+    const amount = 1234.56;
+    const data = [
+      ['cs-CZ', 'CZK', ' Kč', false, '1 234,56 Kč CZK'],
+
+      ['de-AT', 'EUR', '€ ', true, '€ 1 234,56 EUR'],
+      ['de-AT', 'JPY', '¥ ', true, '¥ 1 235 JPY'],
+      ['de-AT', 'OMR', 'OMR ', true, 'OMR 1 234,560'],
+      ['de-AT', 'USD', '$ ', true, '$ 1 234,56 USD'],
+
+      ['en-US', 'EUR', '€ ', true, '€ 1,234.56 EUR'],
+      ['en-US', 'JPY', '¥ ', true, '¥ 1,235 JPY'],
+      ['en-US', 'OMR', 'OMR ', true, 'OMR 1,234.560'],
+      ['en-US', 'USD', '$ ', true, '$ 1,234.56 USD'],
+
+      ['fr-FR', 'EUR', ' €', false, '1 234,56 € EUR'],
+      ['fr-FR', 'JPY', ' JPY', false, '1 235 JPY'],
+      ['fr-FR', 'OMR', ' OMR', false, '1 234,560 OMR'],
+      ['fr-FR', 'USD', ' $US', false, '1 234,56 $ USD'],
+    ];
+
+    it('formats as expected in several locales', () => {
```

Comment:
Same here, let's leverage `it.each` instead

Reply:
Done.   :)